test_doc/book.toml

```diff
@@ -33,6 +33,9 @@ page-ranges = ""
 ignore-invalid-page-ranges = false
 header-template = "<h3 style='font-size:8px; margin-left: 48%' class='title'></h3>"
 footer-template = "<p style='font-size:10px; margin-left: 48%'><span class='pageNumber'></span> / <span class='totalPages'></span></p>"
+## For huge template, you can use the following option to load it from a file.
+# header-template-path = "template/header.html"
+# footer-template-path = "template/footer.html"
```

Comment:
> In order to export my documentation in a specific format expected by my school, I need to include a lot of text and images in the header/footer.

Wow, that's interesting! Actually, I have been asked about the template from file thing before https://github.com/HollowMan6/mdbook-pdf/issues/19#issue-1666170919 I thought Chromium doesn't support rendering complicated header/footer templates. If you are fine with that, can you commit those templates that include text and images directly into this test_doc here (of course remove the sensitive information as well as copyrighted images)? I think those can be a perfect demo for the full version of header/footer template usage, so we can just use `footer-template-path`/`header-template-path` and comment out `footer-template`/`header-template` here:

```suggestion
# header-template = "<h3 style='font-size:8px; margin-left: 48%' class='title'></h3>"
# footer-template = "<p style='font-size:10px; margin-left: 48%'><span class='pageNumber'></span> / <span class='totalPages'></span></p>"
## For huge template, you can use the following option to load it from a file.
header-template-path = "template/header.html"
footer-template-path = "template/footer.html"
```

Thanks so much!

Reply:
@skyf0l